R/workflow.R

```diff
@@ -69,54 +69,56 @@ print.workflow <- function(x, ...) {
 
 print_header <- function(x) {
   if (x$run) {
-    fit <- " [fit]"
+    fit <- " [trained]"
   } else {
     fit <- ""
   }
 
-  cat_line(glue::glue("<workflow{fit}>"))
+  cat(cli::rule(glue::glue("model workflow {fit}")))
```

Comment:
The lack of space between `workflow{` was intentional to not have two spaces in the output when it is trained, could we remove that?